docs/Community/CONTRIBUTING.md

```diff
@@ -1,16 +1,213 @@
+> sharing knowledge and experiences and collaborate we'll be able to produce Best Practices and help to set up Open
+> Source projects in The Netherlands. Because the gap between needs and knowledge in The Netherlands the content of this
+> initiative will be mainly in Dutch.
+>
+> We are very sorry if this creates a feeling of not being inclusive ... which is of course against our [Code of
+> Conduct](CODE_OF_CONDUCT.md); We would like to be as inclusive as possible!
+>
+> BUT to choose to write mainly in Dutch we will be more inclusive to the less equiped and Dutch native readers of our
+> content and those are the primary focus of these practices. Nonetheless, much content is probably not managed here or
+> produced inside these repositories but will be linked to or just translated summaries of content elsewhere.
 
-## Git branching
+Door deze richtlijnen te volgen, communiceer je dat je de tijd respecteert van de ontwikkelaars die dit open
+source-project beheren en ontwikkelen. In ruil daarvoor moeten ze dat respect beantwoorden bij het aanpakken van uw
+melding, het beoordelen van wijzigingen en het helpen afronden van uw pull requests.
 
-## Code reviews
+Houd een open geest! Het verbeteren van documentatie, melden van fouten, of bijdragen aandragen zijn voorbeelden van
+nuttige bijdragen. Veel informatie is mogelijk al ergens beschikbaar, waarschijnlijk in het Engels, en het verwijzen
+naar andere documentatie helpt ons allemaal. Helemaal als daar samenvattingen (of volledige) vertalingen van in het
+Nederlands toegevoegd worden (daar of in dit project)!
 
-## Pull Request Process
\ No newline at end of file
+Mochten bijdragen niet voldoen aan deze richtlijnen dan houden wij ons de vrijheid om commentaren te negeren en
+bijdragen te sluiten. Daarbij zullen wij verwijzen naar deze richtlijnen / Contributing Guide.
+
+> En als je het project leuk vindt, maar gewoon geen tijd hebt om bij te dragen, is dat prima. Er zijn andere eenvoudige
+> manieren om het project te steunen en je waardering te tonen, waar we ook erg blij mee zijn:
+>
+> - Geef het project een ster
+> - Tweet erover
+> - Verwijs naar dit project in de readme van uw project
+> - Noem het project op lokale meetups en vertel het aan je vrienden/collega's
+
+# Basis regels
+
+## Gedragscode
+
+Dit project en iedereen die eraan deelneemt, wordt bestuurd door de [OSPO-NL Gedragscode](CODE_OF_CONDUCT.md). Door deel
+te nemen, wordt van u verwacht dat u zich aan deze code houdt. Gelieve onaanvaardbaar gedrag te melden volgens de
+[Gedragscode#Handhaving](CODE_OF_CONDUCT.md#handhaving).
+
+## Verwachtingen
+
+Vrijwel alle content is beschreven in Markdown. Daarbij maken wij gebruik van MkDocs Material om alle documentatie te
+publiceren. Bij gebruik van plaatjes is het fijn als de bron daarvan ook onderdeel is van dit project ... en bij
+voorkeur in een open formaat. Dat betekent dat deze aangepast en bijgewerkt kunnen worden zonder kosten te maken voor

```

Comment:
Omdat het er zo expliciet staat, moeten we dan aanbevelingen doen? Inkscape/svg voor vectortekeningen, Mermaid.js voor tekst-based afbeeldingen?

Reply:
Tja... open voor suggesties hier. Het helpt wel als we duidelijke voorkeuren hebben, denk ik ... hoewel dat misschien ook kan afschrikken? Ik gebruik graag gemakkelijke tools waarbij het formaat bij voorkeur niet-vendor-specifiek is ... en als ik het niet ken, dan leer ik dat graag 😄 PlantUML gebruik ik ook met regelmaat ...